lib/dradis/plugins/persistant_permissions.rb

```diff
@@ -0,0 +1,42 @@
+module Dradis
+  module Plugins
+    module PersistantPermissions
+      extend ActiveSupport::Concern
+
+      def update
+        @user = User.find(params[:id])
+
+        Permission.transaction do
+          Permission.where(component: self.class.engine_name, user_id: params[:id]).destroy_all
+
+          permissions_params[:permissions]&.each do |permission|
+            # Validate the permission being created is a valid value
+            next unless "::Dradis::Pro::Plugins::#{self.class.engine_name.to_s.classify}::PERMISSIONS".constantize.include?(permission)
+
+            Permission.create!(
+              component: self.class.engine_name,
+              name: permission,
+              user_id: params[:id]
+            )
+          end
+        end
+
+        redirect_to main_app.edit_admin_user_permissions_path(params[:id]), notice: "#{@user.name}'s permissions have been updated."
```

Comment:
@MattBudz @brianp Question about UX. Should we redirect the users to the page with the `anchor` value of the plugin? So that the tab will be active again. Now it's just defaulting to `Projects` tab whenever an admin updates a user's issuelib or remediation tracker permission.

Reply:
Will the anchor actually change the tab?

If so, it would be awesome. but I'm guessing it won't work that simply.